Stripe/STPBankAccount.m

```diff
@@ -24,6 +27,33 @@ @implementation STPBankAccount
 
 @synthesize routingNumber, country, currency, accountHolderName, accountHolderType;
 
+#pragma mark - STPBankAccountStatus
+
++ (NSDictionary<NSString *, NSNumber *> *)stringToStatusMapping {
+    return @{
+             @"new": @(STPBankAccountStatusNew),
```

Comment:
I generally like this dictionary style of doing string <-> enum mappings and we use it a lot of places, but it does somewhat worry me as this method does not create a compiler error if we add a new enum value (whereas doing a switch statement without a default case _does_). 

Switch statements also avoid the "unknown" problem. An exhaustive switch case where we return a string from each possible enum should compile fine I believe? Whereas in this method the compiler will complain that theoretically the map lookup will fail and you could get a nil string returned.

Any feelings on this @bg-stripe ?